networks/README.md

```diff
@@ -11,12 +11,13 @@ WARNING: this information is provided by the community. Please exercise caution
 
 ## Testnet
 
-| Chain ID | Network |                       Name |
-| :------- | ------: | -------------------------: |
-| 11111    |    Fuji |                      WAGMI |
-| 432201   |    Fuji |        Dexalot Fuji Subnet |
-| 335      |    Fuji |  DeFi Kingdoms Fuji Subnet |
-| 808      |    Fuji | Portal Fantasy Fuji Subnet |
+| Chain ID | Network |                        Name |
+| :------- | ------: | --------------------------: |
+| 11111    |    Fuji |                       WAGMI |
+| 432201   |    Fuji |         Dexalot Fuji Subnet |
+| 335      |    Fuji |   DeFi Kingdoms Fuji Subnet |
+| 808      |    Fuji |  Portal Fantasy Fuji Subnet |
+| 8184     |    Fuji | Highrise Blockchain Testnet |
```

Comment:
Have you made a PR to add 8184 to chainlist?

Reply:
Here is the PR https://github.com/ethereum-lists/chains/pull/1579 
Pending approval